tests/acceptance/log-drains/basic-test.js

```diff
@@ -294,6 +305,7 @@ test(`visit ${addLogUrl} and create log to elasticsearch`, function(assert){
     assert.equal(json.drain_password, drainPassword);
     assert.equal(json.drain_username, drainUser);
 
+    json.status = 'provistioning';
     json.id = logDrainId;
     return this.success(json);
   });
```

Comment:
Currently all tests that create a log drain and reload index fail due to the polling
```
Error: Assertion Failed: You cannot reload a record without an ID
```
and 
```
Error: Assertion Failed: You must include an `id` for log-drain in an object passed to `push`
```